src/main/java/org/fundacionjala/coding/raul/movies/Customer.java

```diff
@@ -0,0 +1,47 @@
+package org.fundacionjala.coding.raul.movies;
+
+import java.util.LinkedList;
+
+class Customer {
+    private String name;
+    private LinkedList<Rental> rentals = new LinkedList();
+
+    Customer(final String name) {
+        this.name = name;
+    }
+
+    public void addRental(final Rental arg) {
+
+        rentals.add(arg);
+    }
+
+    public String getName() {
+
+        return name;
+    }
+
+    public String statement() {
+        double totalAmount = 0;
+        int frequentRenterPoints = 0;
+        LinkedList<Rental> rentales = this.rentals;
+        String result = "Rental Record for " + getName() + "\n";
```

Comment:
Consider using a StringBuilder in a for loop

Reply:
Done.